tao_pegtl_vendor/tao_pegtl_vendor-extras.cmake

```diff
@@ -12,5 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-find_package(pegtl REQUIRED)
+# use the fixed version to prevent future compilation issues
+# the number version 2.8.1 is from CMakeLists.txt
+find_package(pegtl 2.8.1 REQUIRED)
```

Comment:
On Ubuntu 22.04, the version tao-pegtl is 2.8.3.   
If want to specified version, we have to put source code of pegtl in our repository like eProsima's  code.    
Or not set last version '.1' (Not sure whether it works. Or set versionMin...[<]versionMax)

Reply:
already to force building 2.8.1 in the `tao_pegtl_vendor` before, and please notice that this tao_pegtl_vendor-extras.cmake is used for other repositories.

Reply:
Whatever the tao-pegtl version is in other platforms, the common_conent_filter will use the pegtl version provided by `tao_pegtl_verdor`. If I didn't set the '.1', there might be a '2.8.9' version that changed some source code in pegtl that caused compilation issues.

Reply:
> already to force building 2.8.1 in the tao_pegtl_vendor before, and please notice that this tao_pegtl_vendor-extras.cmake is used for other repositories.

Yes. You download source code of specified version (Not use system package)

Reply:
It reminded me something about tao-pegtl version, I checked the tao-pegtl(2.8.1) in ubuntu20, so I used this version. I should use the 2.8.3 version that was originally used in fastdds.